airbyte-integrations/connectors/source-facebook-marketing-api-singer/README.md

```diff
@@ -0,0 +1,66 @@
+# Source Facebook Marketing Api Singer
+
+This is the repository for the Facebook Marketing Api source connector, based on a Singer tap.
+For information about how to use this connector within Airbyte, see [the User Documentation](https://docs.airbyte.io/integrations/sources/facebook-marketing-api).
+
+## Local development
+### Build
+First, build the module by running the following from the `airbyte` project root directory: 
+```
+./gradlew :airbyte-integrations:connectors:source-facebook-marketing-api-singer:build
+```
+
+This will generate a virtualenv for this module in `source-facebook-marketing-api-singer/.venv`. Make sure this venv is active in your
+development environment of choice. If you are on the terminal, run the following
+```
+cd airbyte-integrations/connectors/source-facebook-marketing-api # cd into the connector directory
+source .venv/bin/activate
+```
+If you are in an IDE, follow your IDE's instructions to activate the virtualenv. 
+
+**All the instructions below assume you have correctly activated the virtualenv.**.
+
+### Locally running the connector
+```
+python main_dev.py spec
+python main_dev.py check --config sample_files/test_config.json
```

Comment:
i'm fine with `sample_files/test_config.json` existing where it does as a template, but once they put real creds in, should we prefer that we read `--config secrets/config.json`? nbd either way. just a thought.

Reply:
I also prefer running from `secrets` since it's gitignored by default. Changing this and templates to say secrets.